federatedscope/contrib/model/resnet.py

```diff
@@ -0,0 +1,305 @@
+from federatedscope.register import register_model
```

Comment:
If the ResNet is the same with the one in `torchvision.models`, maybe we only need to modify `federatedscope/core/auxiliaries/model_builder.py` rather than copying it here.